sql/core/src/main/java/org/apache/spark/sql/execution/vectorized/OffHeapColumnVector.java

```diff
@@ -97,6 +95,11 @@ public void close() {
     offsetData = 0;
   }
 
+  @Override
+  public void close() {
```

Comment:
this seems like a bug in MIMA... anyway, it's fine to have this workaround for MIMA